packages/react-native/sdks/hermes-engine/hermes-utils.rb

```diff
@@ -203,7 +205,7 @@ def hermestag_file(react_native_path)
 def release_tarball_url(version, build_type)
     # Sample url from Maven:
     # https://repo1.maven.org/maven2/com/facebook/react/react-native-artifacts/0.71.0/react-native-artifacts-0.71.0-hermes-ios-debug.tar.gz
-    return "https://repo1.maven.org/maven2/com/facebook/react/react-native-artifacts/#{version}/react-native-artifacts-#{version}-hermes-ios-#{build_type.to_s}.tar.gz"
+    return "#{REACT_NATIVE_ARTIFACTS_MAVEN_URL}/#{version}/react-native-artifacts-#{version}-hermes-ios-#{build_type.to_s}.tar.gz"
```

Comment:
Is this change necessary? If not, I'd prefer to make it upstream to keep diffs lower

Reply:
reverted